docs/gl_objects/projects.py

```diff
@@ -333,6 +333,22 @@
 pipeline = project.pipelines.create({'ref': 'master'})
 # end pipeline create
 
+# pipeline trigger
+def get_or_create_trigger(project):
```

Comment:
I think this could be  a nice improvement to add this get_or_create to trigger manager